examples/iter/main.neva

```diff
@@ -0,0 +1,19 @@
+const l list<int> = [50, 30, 20, 100]
```

Comment:
nit: could you please name this consistently with other examples like e.g. `19_list_iteration`